libs/datamodel/core/src/transform/ast_to_dml/validation_pipeline/validations/database_name.rs

```diff
@@ -10,7 +8,7 @@ pub(super) fn validate_db_name(
     // How many @ in the error message?
     double_at: bool,
 ) {
-    if let Some(err) = ConstraintNames::is_db_name_too_long(
+    if let Some(err) = datamodel_connector::constraint_names::ConstraintNames::is_db_name_too_long(
```

Comment:
Does the `ConstraintNames` need to be a struct?

Reply:
No, we'll have to change that some day.